catalog/core/catalog-core-directorymonitor/src/test/java/org/codice/ddf/catalog/content/monitor/ContentDirectoryMonitorIT.java

```diff
@@ -0,0 +1,272 @@
+                cxfFeatures(CXF_RT_SECURITY_SAML,
+                        CXF_BINDINGS_SOAP,
+                        CXF_WS_POLICY,
+                        CXF_WS_SECURITY,
+                        CXF_FRONTEND_JAVASCRIPT,
+                        CXF_JAXRS),
+                keystoreAndTruststoreConfig(),
+                initContentDirectoryMonitorConfig());
+    }
+
+    @Override
+    protected List<BundleInfo> bundlesToStart() {
+        List<BundleInfo> bundles = new ArrayList<>();
+        bundles.addAll(testDependencies());
+        bundles.addAll(contentDirectoryMonitorDependencies());
+        return bundles;
+    }
+
+    private List<BundleInfo> testDependencies() {
+        return Arrays.asList(new BundleInfo("org.awaitility", "awaitility"),
+                new BundleInfo("org.mockito", "mockito-core"),
+                new BundleInfo("org.objenesis", "objenesis"));
+    }
+
+    private Option keystoreAndTruststoreConfig() {
+        InputStream keystore = ContentDirectoryMonitorIT.class.getResourceAsStream(
+                "/serverKeystore.jks");
+        InputStream truststore = ContentDirectoryMonitorIT.class.getResourceAsStream(
+                "/serverTruststore.jks");
+
+        return KeystoreTruststoreConfigurator.createKeystoreAndTruststore(keystore, truststore);
+    }
+
+    private Option initContentDirectoryMonitorConfig() {
+        return editConfigurationFilePut(
+                "etc/org.codice.ddf.catalog.content.monitor.ContentDirectoryMonitor-test.cfg",
+                "monitoredDirectoryPath",
+                "");
+    }
+
+    private List<BundleInfo> contentDirectoryMonitorDependencies() {
+        return Arrays.asList(new BundleInfo("org.apache.commons", "commons-lang3"),
+                new BundleInfo("commons-lang", "commons-lang"),
+                new BundleInfo("commons-io", "commons-io"),
+                new BundleInfo("commons-collections", "commons-collections"),
+                new BundleInfo("commons-configuration", "commons-configuration"),
+
+                new BundleInfo("com.google.guava", "guava"),
+                new BundleInfo("org.apache.tika", "tika-core"),
+
+                new BundleInfo("org.apache.shiro", "shiro-core"),
+                new BundleInfo("javax.servlet", "javax.servlet-api"),
+                new BundleInfo("javax.validation", "validation-api"),
+                new BundleInfo("org.bouncycastle", "bcprov-jdk15on"),
+
+                new BundleInfo("ddf.platform.util", "util-uuidgenerator-api"),
+                new BundleInfo("ddf.platform.util", "util-uuidgenerator-impl"),
+                new BundleInfo("ddf.catalog.core", "catalog-core-camelcontext"),
+                new BundleInfo("ddf.platform.api", "platform-api"),
+                new BundleInfo("org.codice.thirdparty", "gt-opengis"),
+                new BundleInfo("ddf.catalog.core", "catalog-core-api"),
+                new BundleInfo("ddf.security.core", "security-core-api"),
```

Comment:
❓ Do we absolutely need to include the security framework in these tests? If this component doesn't do anything special security-wise, we should be able to safely assume security works and not have to pull it in.